app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -213,11 +213,7 @@ export const ContentForm: React.FC<IContentFormProps> = ({
       toast.success(`"${contentResult.headline}" has successfully been saved.`);
 
       if (!originalId)
-        navigate(
-          `${getContentPath(contentResult?.contentType)}${combined ? 'combined/' : ''}${
-            contentResult.id
-          }`,
-        );
+        navigate(getContentPath(combined, contentResult.id, contentResult?.contentType));
```

Comment:
Navigation had a number of bugs depending on whether the user was using the combined view or not.  I've moved all the logic to the function.